content/sandbox.markdown

```diff
@@ -10,20 +10,36 @@ excerpt: Welcome to DNSimple. This page explains our sandbox environment. Enjoy
 
 ---
 
-We maintain an isolated test site at [sandbox.dnsimple.com](https://sandbox.dnsimple.com/). Please feel free to test your applications that integrate with the DNSimple API here.
+We maintain an isolated test site at [sandbox.dnsimple.com](https://sandbox.dnsimple.com/). Please feel invited to test your applications that integrate with the DNSimple API here.
+
+### Disclaimer
 
 Sandbox data and systems are generally treated like production, however we may not apply the same backup, monitoring and high availability policies. That means we may occasionally need to take this environment down, wipe out a portion, or clear all stored data.
 
+## Getting started
+
+The DNSimple sandbox is a independent system, you have to create another account and register some domains to test your integrations.
 
-## URL
+- Go to [sandbox.dnsimple.com/signup](https://sandbox.dnsimple.com/signup) and register your account. _It does not have to match your DNSimple live account._
+- When asked to choose a plan choose `Skip this step and choose a plan later` at the bottom. If you want to test features that requires a valid subscription please refer to the [Testing Subscriptions](#testing-subscriptions) section.
+
+### URL
 
 The API hostname is `api.sandbox.dnsimple.com`. This hostname follows the same [URL and version](/overview/#url) conventions of the production environment.
 
-## Special Settings
+### Sandboxing 3rd parties
+
+This site is generally an exact duplicate of the production application, but there are some exceptions. Instead of actually performing domain registration, charging credit cards, etc. it _mocks_ these functions.
+
+
+## Testing Domains
```

Comment:
I try to figure out what the sandbox actually can do with certs and will add it then :)